test/sqllogictest/tpch.slt

```diff
@@ -1134,14 +1134,12 @@ ORDER BY
 
 %1 =
 | Get materialize.public.revenue (u27)
-| Filter !(isnull(#1))
 | ArrangeBy (#0)
 
 %2 =
 | Get materialize.public.revenue (u27)
 | Reduce group=()
 | | agg max(#1)
-| Filter !(isnull(#0))
 | ArrangeBy (#0)
 
 %3 =
```

Comment:
To explain what is going on here: the `revenue` view is a `GROUP BY` aggregation with a key, so we can determine that the associated value cannot be null.
```
create view revenue (supplier_no, total_revenue) as
    SELECT
        l_suppkey,
        sum(l_extendedprice * (1 - l_discount))
    FROM
        lineitem
    WHERE
        l_shipdate >= DATE '1996-01-01'
        AND l_shipdate < DATE '1996-01-01' + INTERVAL '3' month
    GROUP BY
        l_suppkey;
```
At least, I think that is the line of reasoning.

Reply:
Or, I suppose we can determine this under the presumption that `l_extendedprice` and `l_discount` are not null? I'm not actually certain, now that I type that out. Will check it out more.

Reply:
Yes, can confirm that in `tpch.slt` we assert `NOT NULL` for each.